agent/credentials/providers/credentials_filename_windows.go

```diff
@@ -0,0 +1,24 @@
+//go:build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package providers
+
+import "path/filepath"
+
+var (
+	// defaultRotatingCredentialsFilename is the default location of the credentials file
+	// for RotatingSharedCredentialsProvider.
+	defaultRotatingCredentialsFilename = filepath.Join("C:\\Windows\\System32\\config\\systemprofile\\.aws", "credentials")
```

Comment:
You should just write the full path here. We hard coded everything so just make it a const.

Reply:
Sure thing. Will update this.